.github/workflows/docker-build.yaml

```diff
@@ -463,6 +463,44 @@ jobs:
             -   name: Cypress container logs
                 if: ${{ failure() }}
                 run: docker compose logs cypress
+    translations-dump-check:
+        name: Check all translations are dumped
+        needs: build-php-fpm-image
+        if: |
+            always() && !failure() && !cancelled() &&
+            (
+                github.event.pull_request.head.repo.full_name == 'shopsys/shopsys' ||
+                github.ref_protected == true
+            ) &&
+            (needs.build-php-fpm-image.result == 'success' || needs.build-php-fpm-image.result == 'skipped')
+        runs-on: ubuntu-22.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Download docker-compose.yml from artifacts
+                uses: actions/download-artifact@v3
+                with:
+                    name: docker-compose
+            -   name: Build application
+                run: docker compose up -d php-fpm storefront
+            -   name: Check translations are dumped
+                run: |
+                    docker compose exec -T php-fpm composer install --optimize-autoloader --no-interaction
+                    docker compose exec -T php-fpm php phing translations-dump
+                    docker compose cp php-fpm:/var/www/html/ .
+                    cp -a ./html/. ./
+                    rm -rf ./html
```

Comment:
I assume this is due to fact that we don't have a volume in the docker-compose file? 
Can you explain please the reason for second cp and rm?

Reply:
We have discussed this via Slack. There is no way to copy only the content of the directory using docker compose cp, so it is necessary to do it like this.